src/frontend/components/DashboardDocument/index.tsx

```diff
@@ -138,6 +150,18 @@ const DashboardDocument = (props: DashboardDocumentProps) => {
           />
         </DashboardDocumentInnerContainer>
       );
+    default:
+      return (
+        <DashboardDocumentInnerContainer>
+          {commonStatusLine}
```

Comment:
I'll used a functional component for `commonStatusLine` by reading this line, I believed that was a prop or another variable. As it is a React component, you should use `jsx` here.
`<CommonStatusLine />`

Reply:
Wehn I try I have this error : `Property 'commonStatusLine' does not exist on type 'JSX.IntrinsicElements'.`

`commonStatusLine ` is not a component, it's a variable made to avoid repetition.

Reply:
You have this error because you do not use a Capitalized name for this component so JSX.

Reply:
I made the change to use a component here. I would appreciate if you take a look of what I made. :pray: